test/colony-network-recovery.js

```diff
@@ -211,8 +211,8 @@ contract("Colony Network Recovery", accounts => {
 
       await colonyNetwork.enterRecoveryMode();
 
-      await colonyNetwork.setStorageSlotRecovery(19, "0x02");
-      await colonyNetwork.setStorageSlotRecovery(20, `0x${new BN(7).toString(16, 64)}`);
+      await colonyNetwork.setStorageSlotRecovery(20, "0x02");
```

Comment:
Fixing these failures can't have been fun... 😭

Reply:
Alex did the hard work on finding out the exact storage slots in he first place, here I just added 1 to the slots positioned _after_ the one new parameter I introduced.